community.html

```diff
@@ -20,19 +20,24 @@
             <p class="supportparagraph">Please feel free to ask us questions about Project Jupyter!</p>
             <div class="col-md-12">
                 <div class="col-md-4 material material-1">
-                   <p class="community-resource"><a href="http://stackoverflow.com/questions/tagged/jupyter">Jupyter on Stack Overflow</a><p>
+                   <p class="card-header">Jupyter on Stack Overflow<p>
```

Comment:
@cameronoelsen Double check the copy pasta on the card-header lines. Looks like some should close with </p>